scripts/initialize_docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -e
+
+docker login
+docker buildx create --use --name stork-cli-builder --driver docker-container
```

Comment:
we run this once initially to log in to dockerhub and to set up the multi-platform build context